spec/unleash/bootstrap/from_uri_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'unleash/bootstrap/from_uri'
+require 'json'
+
+RSpec.describe Unleash::Bootstrap::FromUri do
+  it 'loads bootstrap toggle correctly from URL' do
+    bootstrap_file = './spec/unleash/bootstrap-resources/features-v1.json'
```

Comment:
maybe consider in-lining the JSON here. The data is not really re-used in different contexts.